content/en/agent/kubernetes/integrations.md

```diff
@@ -338,6 +360,25 @@ Notice that each of the three values is a list. Autodiscovery assembles list ite
 
 Unlike auto-conf files, **key-value stores may use the short OR long image name as container identifiers**, e.g. `redis` OR `redis:latest`.
 
+[1]: /agent/faq/template_variables/
+{{% /tab %}}
+{{% tab "Helm" %}}
+
+The following configuration defines the integration template for `redis` containers with a custom password parameter:
+```yaml
+  confd
+    redisdb.yaml: |-
+      ad_identifiers:
+        - redis
+      init_config:
+      instances:
+        - host: %%host%%
+          port: 6379
+          password: %%env_REDIS_PASSWORD%%
+```
+As a result, the agent will contain a `redis.yaml` file with the above configuration in the `/confd` directory.
+**Note**: The `"%%env_<ENV_VAR>%%"` template variable logic is used to avoid storing the password in plain text, hence the `REDIS_PASSWORD` environment variable must be passed to the Agent. See the [Autodiscovery template variable documentation][1].
```

Comment:
```suggestion
**Note**: The `"%%env_<ENV_VAR>%%"` template variable logic is used to avoid storing the password in plain text. Hence, you must pass the `REDIS_PASSWORD` environment variable to the Agent. See the [Autodiscovery template variable documentation][1].
```